src/amber/controller/helpers/asset.cr

```diff
@@ -0,0 +1,44 @@
+module Amber::Controller::Helpers
+  module Asset
+    ASSET_MANIFEST = {} of String => String
+
+    {{ run "../../run_macros/generate_asset_helpers" }}
+
+    macro asset_path(path)
+      {% if path.is_a?(StringLiteral) %}
+        {% if Amber::Controller::Helpers::Asset::ASSET_MANIFEST[path] %}
+          {{ "/dist" + Amber::Controller::Helpers::Asset::ASSET_MANIFEST[path] }}
```

Comment:
`/dist` is consistent with the current implementation and is a fine default, but should this be something that can be overridden?

Reply:
I'm up for moving it into configuration.  I don't know where/how to do configuration settings though.  Point me in the right direction and I can go from there.

Reply:
You should add a new setting to src//amber/environment/settings.cr . Then it becomes available for use in .yml files and in Amber::Server.configure {} block.

Reply:
How do I use a config variable inside of a macro?  I can't get it to compile.

Reply:
Nevermind.  I think I figured it out :).